packages/custom/icu/public/components/officeDocument-list/officeDocument-list.js

```diff
@@ -24,7 +24,18 @@ function OfficeDocumentListController($scope, $state, BoldedService, NotifyingSe
             result.created = new Date(result.created);
             $scope.items.push(result);
             return result;
-        });
+        })
+        .then(item => {
+            if(parent && parent.type === 'task'){
+                TasksService.getById(context.entityId)
+                    .then(parentEntity => {
+                        parentEntity.officeDocuments = parentEntity.tasks || [];
+                        parentEntity.officeDocuments.push(item._id);
+                        TasksService.update(parentEntity._id, parentEntity);
```

Comment:
This seems to be duplicate of the function `addToParent` in `packages/custom/icu/server/controllers/documents.js`

Also, the reference `parentEntity._id` is not treated as expected in the `TasksService.update` function, and causes a 404 response status.